### DISCUSSION TITLE
driver.click() works but driver.uc_click() doesn't

### DISCUSSION THREAD

**Discussion by DataIDraw** [seleniumbase/SeleniumBase#2745] [Q&A] [answered]
Hi,

Before all, thank a lot @mdmintz for all your work and help !

Quick question, about driver.click() and driver.uc_click(). I tried the following code :

`driver.uc_click("#client > div > div.cards > div:nth-child(2)")`

I don't have any error in the log but I expected a "click button" and it doesn't click. But if I try the following code : 

`driver.click("#client > div > div.cards > div:nth-child(2)")`

It works, the button is clicked.

Did you ever see this behaviour with uc_click() ?

Thanks a lot !

**Comment by mdmintz** [comment, marked as answer]
I haven't seen that behavior. `uc_click(selector)` should be used to bypass detection, and that selector doesn't look like an element that would necessarily check for Selenium when clicked. Most of the time, it's a `span` element, but it can also be `button` or `a`.

Check to see if a console `document.querySelector(selector).click()` clicks the element for you. That's how the click would happen.

**Reply by SAKMZ** [reply]
Hi,

Thank you so much for all the amazing work you do.

I had worked on a project in Feb, I just updated the seleniumbase version to the latest and ran the script again, and the uc_click doesn't work anymore. But the strange thing is when I downgrade to a lower version (tested on 4.25.0) it works again, but not on the latest version. Do you know what might be the issue?

Thanks in advance!

**Reply by mdmintz** [reply]
Hi, I haven't seen that. The current UC Mode examples are still working for me.

**Reply by CJohnno77** [reply]
I also had the same issue. click() worked but uc_click() didn't work.  In the browser console, elem.click() **didn't** work.  elem is the element I wanted to click. In the console, I got elem by doing: elem = $x(xpath expression)[0]

**Reply by mdmintz** [reply]
The JS methods require CSS Selectors, not XPath.
